tests/test_models/test_return_data_object.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+"""
+Contains the testing module for returnDTO
+"""
+from models.return_data_object import ReturnDataTransferObject
+import unittest
+
+
+class TestReturnDTO(unittest.TestCase):
```

Comment:
I have some global comments about the way you are testing here. First, what you have done is working and fine, I just want to enlarge the debate and thought.

You are using the method `setUp` to set up your testing environment, in this case you are creating an empty object.

Then after, you are trying to set this object with some dummy data, often the same:
```
{'eventList': None, 'weatherInformation': None}
```

I will suggest you, for more readibilty and make it easier to follow for the reader, to set those dummy data in the `setUp` method as well:
```
def setUp(self):
        """set up for all unittests"""
        self.testReturnObject = ReturnDataTransferObject()
        self.objectWithNoneEventListWeather = {'eventList': None, 'weatherInformation': None}
        self.objectWithOneEventListNoneWeather = {'eventList': ["test", "object"], 'weatherInformation': None}
        self.objectWithNoneEventListOneWeather = {'eventList': None, 'weatherInformation': 123}
        ...
```

Then later in your code, you are using those object. By doing like this, the reader is reading `setUp`, understand your dummy object, and doesn't need to come back to them after. It will make more sense when you have more testing data.

When I read the test (the method starting with `test_`), I need to think and focus on the test itself, not decrypting your dummy data.